sdk/marketplaceordering/azure-resourcemanager-marketplaceordering/CHANGELOG.md

```diff
@@ -1,14 +1,35 @@
 # Release History
 
-## 1.0.0-beta.3 (Unreleased)
+## 1.0.0-beta.3 (2024-10-09)
 
-### Features Added
+- Azure Resource Manager MarketplaceOrdering client library for Java. This package contains Microsoft Azure SDK for MarketplaceOrdering Management SDK. REST API for MarketplaceOrdering Agreements. Package tag package-2021-01-01. For documentation on how to use this package, please see [Azure Management Libraries for Java](https://aka.ms/azsdk/java/mgmt).
 
 ### Breaking Changes
 
-### Bugs Fixed
+#### `models.MarketplaceAgreements` was modified
+
+* `models.AgreementTerms cancel(java.lang.String,java.lang.String,java.lang.String)` -> `models.OldAgreementTerms cancel(java.lang.String,java.lang.String,java.lang.String)`
+* `models.AgreementTerms getAgreement(java.lang.String,java.lang.String,java.lang.String)` -> `models.OldAgreementTerms getAgreement(java.lang.String,java.lang.String,java.lang.String)`
+* `java.util.List list()` -> `models.OldAgreementTermsList list()`
```

Comment:
this is expected: https://github.com/Azure/azure-rest-api-specs/commit/bef7ff1b9b0629adbc75e19bebc755af43930d83